src/js/view/controlbar.js

```diff
@@ -90,7 +126,7 @@ define([
                 mute: muteButton,
                 volume: volumeSlider,
                 volumetooltip: volumeTooltip,
-                cast: button('jw-icon-cast jw-off', this._api.castToggle, this._localization.cast),
+                cast: !utils.isSafari()? castButton(this._localization.cast) : button('jw-icon-airplay jw-off', this._api.castToggle, this._localization.airplay),
```

Comment:
How did we handle the fullscreen element in Safari iOS before we could do inline?

If we just want to make sure it never shows the casting element, we can flip the logic to only create the chromecast element for non-iOS Chrome, otherwise it will create an airplay element that will always be hidden for non-Safari browsers. 

If we don't want to generate the element in unsupported browsers, I can rework the logic to do so.